client/src/common/components/DropdownMenu.jsx

```diff
@@ -77,7 +84,7 @@ DropdownMenu.propTypes = {
       icon: PropTypes.string,
       label: PropTypes.string,
 
-      callback: PropTypes.func
+      onClick: PropTypes.func
```

Comment:
this should be required - if it is undefined and item clicked then it will try to run undefined